community.md

```diff
@@ -0,0 +1,11 @@
+
+## Community
+
+
+
+*   [Contribute](https://thegooddocsproject.groups.io/g/contribute.md)
```

Comment:
Have you checked these links in a rendered website? I haven't but suspect that contribute.html will find the correct page, but contribute.md will get a 404.

Reply:
Was unable to test in a rendered website, and so followed existing code. The link has now been changed accordingly.